bootstrap_test.go

```diff
@@ -44,7 +44,6 @@ func (s *testBootstrapSuite) TestBootstrap(c *C) {
 	defer testleak.AfterTest(c)()
 	store, dom := newStoreWithBootstrap(c, s.dbName)
 	defer dom.Close()
-	defer store.Close()
```

Comment:
why remove this ?

Reply:
because store has been closed in line 97